source/settings/plugin/fts-solr-plugin.rst

```diff
@@ -12,69 +12,68 @@ Settings
 .. dovecot_plugin:setting:: fts_solr
    :plugin: fts-solr
    :values: @string
+   :removed: 2.4.0,3.0.0
```

Comment:
The assumption should be that almost every setting will change for 2.4.0/3.0.0.  I would rather have a single warning/info box at the top of the page that mentions this with a link to 2.3 documentation (i.e. we should have a doc.dovecot.org/2.3/ permalink).
3.0 (and 2.4) is the chance to kind of start over and clean up all this stuff in the documentation.  We don't need to worry about BC stuff, because there isn't any if we are demanding config rewrite.